prereise/gather/demanddata/transportation_electrification/immediate.py

```diff
@@ -0,0 +1,184 @@
+# DELETED: comm_type, LOAD, vmtsc
+
+import prereise/gather/demanddata/transportation_electrification/mileage
+import prereise/gather/demanddata/transportation_electrification/charging
+import numpy as np
+
+
+def immediate_charging(
+    census_region, model_year, veh_range, kwhmi, power, location_strategy
+):
+    """Immediate charging function
+
+    :param int census_region: any of the 9 census regions defined by US census bureau.
+    :param int model_year: year that is being modelled/projected to, 2017, 2030, 2040, 2050.
+    :param int veh_range: 100, 200, or 300, represents how far vehicle can travel on single charge.
+    :param int kwhmi: fuel efficiency, should vary based on vehicle type and model_year.
+    :param int power: charger power, EVSE kW.
+    :param int location_strategy: where the vehicle can charge-1, 2, 3, 4, or 5; 1-home only, 2-home and work related, 3-anywhere if possibile, 4-home and school only, 5-home and work and school.
+    :return: (*numpy.ndarray*) -- charging profiles.

```

Comment:
Is the output of this function one profile, or multiple? It looks like the result is a 1-D array that's 8760 or 8784 entries long, depending on whether it's a leap year or not.